SecurityPrivacyQuestionnaire.md

```diff
@@ -0,0 +1,184 @@
+controlled environment.
+
+## [2.4 How do the features in your specification deal with sensitive information?](https://www.w3.org/TR/security-privacy-questionnaire/#sensitive-data)
+
+Controlled Frame is only available to isolated contexts (TODO: link) and is not
+available outside of isolated contexts. In addition, the Controlled Frame API is
+only available if the "controlled-frame" permissions policy is requested and
+granted by the user agent's IWA environment.
+
+If Controlled Frame's requirements are met and it's available, the specification's
+API surface will be available. Each method is responsibly designed to handle just
+the data, identifiers, and arguments that it requires.
+
+## [2.5 Does data exposed by your specification carry related but distinct information that may not be obvious to users?](https://www.w3.org/TR/security-privacy-questionnaire/#hidden-data)
+
+Controlled Frame is only available to isolated contexts (TODO: link). In Chrome,
+isolated contexts are usually only available to IWAs (TODO: link).
+
+IWAs support all of the APIs available to the web, similar to PWAs. IWAs require
+that any permission required to be granted by the user agent must be specified
+in the high watermark permission manifest's "permissions_policy".
+
+As for the Controlled Frame API, specifically, as an embedding control, it is
+possible to use it in multiple ways where interaction with the API could expose
+distinct information that may not be obvious to users.
+
+Examples of how data could be exposed that may not be obvious to users:
+
+* If the Controlled Frame is used similar to &lt;iframe&gt;, all concerns
+about how an &lt;iframe&gt; may obfuscate the origin source apply to Controlled
+Frame. The IWA will need to take care to ensure that users understand the source
+of the embedded content before interacting with it.
+
+* If users interact with embedded content within a Controlled Frame, the
+embedder could exfiltrate that interaction via JavaScript.
+
+* An embedded could interact with embedded content and use APIs that carry
+some associated exposure risk.
+
+## [2.6 Do the features in your specification introduce state that persists across browsing sessions?](https://www.w3.org/TR/security-privacy-questionnaire/#persistent-origin-specific-state)
+
+Controlled Frame is only available to isolated contexts (TODO: link).
+
+The methods in the specification currently do not store state that perists
+across browsing sessions. As an embedding control, though, Controlled Frame
+allows state to be stored across browsing sessions for any origin that is loaded
+within it.
+
+This is a key reason why Controlled Frame is only available to isolated
+contexts.  Isolated contexts store their data in a separate storage partition
+from the default storage partition used in normal web browsing. Any data stored
+in a way that persists will be stored in this separate storage partition.
```

Comment:
Maybe clarify that the storage partition will be Controlled Frame specific? It's not actually the same storage partition that the IWA itself uses.

Reply:
Done.